content/en/docs/components/trainer/operator-guides/migration.md

```diff
@@ -23,6 +23,89 @@ The key features introduced by Kubeflow Trainer are:
 - Enhanced MPI support: featuring MPI-Operator V2 features with SSH-based optimization to boost
   MPI performance.
 
-## Migration Paths
+## Migration paths
+
+Kubeflow Trainer v2 introduces new APIs that replace the older, framework-specific CRDs such as `PyTorchJob`, `TFJob`, and `MPIJob`. These new APIs—`TrainingRuntime`, `TrainJob`, and `ClusterTrainingRuntime`—offer a more flexible and unified interface for defining training jobs across frameworks.
+
+This section shows how to migrate an existing PyTorch job to the new API.
+
+### Migrate PyTorchJob to TrainingRuntime
+
+The following example demonstrates how to migrate from a `PyTorchJob` (v1) to a `TrainingRuntime` (v2alpha1).
+
+### Old: PyTorchJob (v1)
+```yaml
+apiVersion: kubeflow.org/v1
+kind: PyTorchJob
+metadata:
+  name: pytorch-simple
+  namespace: kubeflow
+spec:
+  pytorchReplicaSpecs:
+    Master:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
+              command: ["python3", "/opt/pytorch-mnist/mnist.py", "--epochs=1"]
+    Worker:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
+              command: ["python3", "/opt/pytorch-mnist/mnist.py", "--epochs=1"]
+```
+### New: TrainingRuntime (v2alpha1)
+```yaml
+apiVersion: trainer.kubeflow.org/v2alpha1
+kind: TrainingRuntime
+metadata:
+  name: torch-distributed-multi-node
+spec:
+  mpiPolicy:
+    numNodes: 2
+  template:
+    spec:
+      replicatedJobs:
+        - name: node
+          template:
+            metadata:
+              labels:
+                trainer.kubeflow.org/trainjob-ancestor-step: trainer
+            spec:
+              containers:
+                - name: trainer
+                  image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
+                  env:
+                    - name: MASTER_ADDR
+                      value: "pytorch-node-0-0.pytorch"
+                    - name: MASTER_PORT
+                      value: "29400"
+                  command: ["torchrun", "train.py"]
+```
+The new `TrainingRuntime` CRD allows you to define reusable templates and use MPI-based distributed training with greater flexibility. The `replicatedJobs` and `mpiPolicy` fields replace framework-specific roles like `Master` and `Worker`.
```

Comment:
MPI is used only for ML frameworks that require MPI functionality (e.g. DeepSpeed). So it is not needed for `torchrun`.